device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D24C52/sai_4600c_24x50g_52x100g.xml

```diff
@@ -0,0 +1,501 @@
+
+        <!-- ISSU enabled -->
+        <issu-enabled>1</issu-enabled>
+
+        <!-- Number of ports in the following port list -->
+        <number-of-physical-ports>64</number-of-physical-ports>
+
+        <!-- Global port late create -->
+        <late-create-all-ports>1</late-create-all-ports>
+
+        <!-- List of ports in the device -->
+        <ports-list>
+            <port-info>
+                <local-port>105</local-port>
+                <width>4</width>
+                <module>0</module>
+
+                <!-- 0 none, 1=2, 2=4, 3=2,4 -->
+                <breakout-modes>3</breakout-modes>
+
+                <!-- (BITMASK) 2 - 1Gb , 16 - 10Gb , 32 - 40Gb , 384 - 50Gb , 1536 - 100Gb , 1536 - 200Gb -->
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>107</local-port>
+                <width>4</width>
+                <module>1</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>109</local-port>
+                <width>4</width>
+                <module>2</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>111</local-port>
+                <width>4</width>
+                <module>3</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>97</local-port>
+                <width>4</width>
+                <module>4</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>99</local-port>
+                <width>4</width>
+                <module>5</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>101</local-port>
+                <width>4</width>
+                <module>6</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>103</local-port>
+                <width>4</width>
+                <module>7</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>1536</port-speed>
+            </port-info>
+            <port-info>
+                <local-port>121</local-port>
+                <split>2</split>
+                <width>4</width>
+                <module>8</module>
+                <breakout-modes>3</breakout-modes>
+                <port-speed>384</port-speed>
```

Comment:
is it expected that i see 384 here, I compared against https://github.com/sonic-net/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D24C52/sai_3800_24x50g_52x100g.xml , In this i see that all ports are set to port speed 1536

Reply:
@anish-n The values are: `1536 - 100G`, `384 - 50G`. The SKU you are referring to probably has a bug. Despite of that, this SKU has `port late create` enabled, meaning all ports are managed directly by SONiC, so SAI config actually does not matter

Reply:
ok thanks